ignore presence/absence of xsltproc, just use the bundled manuals
which are included in tarball downloads.

proper fix would use the pre-built manpages directly if present, and only
run xsltproc if they don't exist or are outdated, but I haven't hit on the
right cmake magic to do this.

Index: man/CMakeLists.txt
--- man/CMakeLists.txt.orig
+++ man/CMakeLists.txt
@@ -5,7 +5,7 @@
 # because the install is optional.
 
 find_program(XSLTPROC xsltproc OPTIONAL)
-if(XSLTPROC AND NOT WIN32)
+if(FALSE)
 	function(compile_manpage page)
 		add_custom_command(OUTPUT ${PROJECT_SOURCE_DIR}/man/${page}
 			COMMAND xsltproc --xinclude --nonet ${PROJECT_SOURCE_DIR}/man/${page}.xml -o ${PROJECT_SOURCE_DIR}/man/
@@ -27,7 +27,11 @@ if(XSLTPROC AND NOT WIN32)
 	compile_manpage("mosquitto_signal.1")
 	compile_manpage("mosquitto_sub.1")
 	compile_manpage("mqtt.7")
+elseif()
+	message(WARNING "xsltproc not found: manpages cannot be built")
+endif()
 
+if(TRUE)
 	install(FILES
 		mosquitto_ctrl.1
 		mosquitto_ctrl_dynsec.1
@@ -49,9 +53,4 @@ if(XSLTPROC AND NOT WIN32)
 		DESTINATION ${CMAKE_INSTALL_MANDIR}/man7 OPTIONAL
 	)
 	install(FILES mosquitto.8 DESTINATION ${CMAKE_INSTALL_MANDIR}/man8 OPTIONAL)
-
-elseif(WIN32)
-	message(WARNING "xsltproc not found: manpages cannot be built")
-else()
-	message(FATAL_ERROR "xsltproc not found: manpages cannot be built")
 endif()
